k8s/grafana/Makefile

```diff
@@ -1,19 +1,26 @@
-include ../app.Makefile
 include ../crd.Makefile
 include ../gcloud.Makefile
 include ../var.Makefile
 
-TAG ?= 5.3
+# The digest should be updated automatically, or tag <major>.<minor>.<path/build>
+# should be available on GCP Marketplace.
+# Use `.build/grafana/show-version` target to fill the RELEASE variable.
+GRAFANA_TAG ?= sha256:2d61c53e58b27e59cd77c9899208bfa31a2d12d27822a3cc3ef362213141a406
 METRICS_EXPORTER_TAG ?= v0.5.1
 
-$(info ---- TAG = $(TAG))
+TIMESTAMP := $(shell date +%s)
```

Comment:
I think this should be only relevant in the actual release build. We shouldn't generate this value every time when we build locally. Fine for now but should follow up to harden this.